src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="action-attr"
+    method="post"
+    data-qa="big"
+    class="bar"
+    >
+      <div class="fon fon-big">
+        <img
+        src="images\Search.svg"
+        alt="icon"
+        class="bigIcon"
+        >
+        <input
+        type="text"
+        name="bigbar"
+        id="bigbar"
+        class="input inputBig"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </div>
```

Comment:
why do you need this `div `container? could you make the same with `form`?

Reply:
div was deleted